Sources/Public/Animation/LottiePlaybackMode.swift

```diff
@@ -3,115 +3,155 @@
 
 import Foundation
 
+// MARK: - LottieMarkerPosition
+
+/// The position within a marker.
+public enum LottieMarkerPosition: Hashable {
```

Comment:
Could you move this to the bottom of the file below `LottiePlaybackMode`? Ideally `LottiePlaybackMode` would be the first type in this file